doc/api-extensions.md

```diff
@@ -2306,3 +2306,9 @@ during instance initialization using the `--device` flag.
 
 Note that these configuration are applied only at the time of instance creation and subsequent modifications have
 no effect on existing devices.
+
+## `client_operation_wait`
```

Comment:
Lets call this `agent_operation_wait` as the server-side /wait endpoint has been around in some form since at least LXD 3.13 and so we can safely consider this always available. Whereas the agent /wait endpoint is brand new.

Reply:
hrm although that will mean the client talking to older servers will have to check for `agent_operation_wait` which wont make sense.

OK lets call it `operation_wait` to align with `operations_get_query_all_projects`.

This way only newer clients talking to newer servers will use this approach, which will also ensure that any existing timeout issues with the /wait endpoint are resolved before clients start using it.

Reply:
Done.